### ISSUE TITLE
Lock the app

### ISSUE THREAD

**Issue by brandonhaslegs** [radicle-dev/radicle-upstream#2127]
There should be a way to lock the app (without quitting and restarting it).

Pressing this button should take the user to the password input screen.

**Comment by geigerzaehler** [comment]
This is blocked by missing functionality in `radicle_daemon`. It’s also unclear how locking and password management will involve in light of [RFC 682](https://github.com/radicle-dev/radicle-link/blob/master/docs/rfc/0682-application-architecture.adoc) in radicle-link. I suggest we put this on hold for now and wait until there’s a need to implement this from the protocol side.

**Comment by brandonhaslegs** [comment]
Just meant to log it, don't think it's high priority.

**Comment by geigerzaehler** [comment]
@Kaihuang724 I don’t think this is relevant anymore. It’s unclear why we want this and how this would work and interact with the rest of the system. Until we see a clear need for this we can close it.
